app/build.gradle.kts

```
@@ -36,7 +36,17 @@ kapt {
 }
 
 val privateProperties = Properties().apply {
-    load(FileInputStream("privateConfig/private.properties"))
+    try {
+        load(FileInputStream("privateConfig/private.properties"))
+    } catch (e: Exception) {
```

I suggest catching only the specific exception this code intends to handle:
```suggestion
    } catch (e: FileNotFoundException) {
```

This way it won't mask other problems: for example the file exists but is incorrectly formatted and `load(...)` throws some parsing exception.

I also suggest logging the caught exception, again to be prudent.

Up to you and the maintainers/reviewers, I have no authority or knowledge of this project :)

---

Yes, limiting the caught exception is always a good choice. Logging is not mandatory here

---

This PR has been merged, as no updated has been received from the author and we decided that the requirements are satified